zubhub_frontend/zubhub/src/assets/js/styles/views/page_wrapper/pageWrapperStyles.js

```diff
@@ -7,7 +7,7 @@ const styles = theme => ({
   },
   input: {
     flex: 1,
-    marginRight: '2rem',
+    marginRight: '2.8rem',
```

Comment:
I've a suggestion here with respect to values with such values (decimal values), can we have it uniform as positive integers?

Reply:
These are `rems` @yokwejuste